mirdata/datasets/giantsteps_key.py

```diff
@@ -112,13 +112,13 @@ def __init__(
             metadata,
         )
 
-        self.audio_path = os.path.join(self._data_home, self._track_paths["audio"][0])
-        self.keys_path = os.path.join(self._data_home, self._track_paths["key"][0])
+        self.keys_path = self.get_path("key")
         self.metadata_path = (
-            os.path.join(self._data_home, self._track_paths["meta"][0])
-            if self._track_paths["meta"][0] is not None
-            else None
+            self.get_path("meta") if self._track_paths["meta"][0] is not None else None
```

Comment:
shouldn't the `none_path_join` remove the need for the if/else? i.e. i think you can directly call `self.metadata_path = self.get_path("meta")` and the result will be the same